code/style.css

```diff
@@ -1,58 +1,169 @@
+header {
+  font-family: "Candal";
```

Comment:
Let's not forget about font fallback - if by any chance your `Candal` font wouldn't be loaded, we are left with basic, browser-implemented font. If on the other hand you would add font fallback, that font would be used in case of problem with first one :

```
font-family: "Candal", Helvetica;
```